lib/rucio/rse/protocols/xrootd.py

```diff
@@ -114,16 +114,14 @@ def stat(self, path):
             self.logger.info('xrootd.stat: filesize cmd: {}'.format(cmd))
             status_stat, out, err = execute(cmd)
             if status_stat == 0:
-                ret['filesize'] = out.split(b'\n')[2].split()[-1].decode()
+                ret['filesize'] = out.split('\n')[2].split()[-1]
```

Comment:
why did you change this here? i remember adding this explicitly to keep compatible with some clients

Reply:
Hey @mlassnig,
When I run the test_rse_protocol_xrootd tests (in dev/[Dockerfile_py3](https://github.com/rucio/containers/blob/master/dev/Dockerfile_py3) container) with 

```
ret['filesize'] = out.split(b'\n')[2].split()[-1].decode()
```

The following tests fail:
```
lib/rucio/tests/test_rse_protocol_xrootd.py::TestRseXROOTD::test_put_mgr_ok_multi FAILED                                                                                                             [ 62%]
lib/rucio/tests/test_rse_protocol_xrootd.py::TestRseXROOTD::test_put_mgr_ok_single FAILED
```

The stack trace is as follows:
```
            status_stat, out, err = execute(cmd)
            if status_stat == 0:
>               ret['filesize'] = out.split(b'\n')[2].split()[-1].decode()
E               TypeError: must be str or None, not bytes

lib/rucio/rse/protocols/xrootd.py:117: TypeError

```
Therefore, I had changed the separator type to `str`